device_name: "/dev/input/event27"
max_force: 1.0
min_force: 0.2
update_rate: 0.1
